src/backend/opengl/CommandBufferGL.cpp

```diff
@@ -26,6 +27,8 @@
 namespace backend {
 namespace opengl {
 
+    PersistentPipelineState persistentPipelineState;
```

Comment:
Also at the beginning of Execute, we should probably reset all the states to the defaults.